src/components/tableOfContents/js/TableOfContents.js

```diff
@@ -112,7 +112,7 @@ var fluid_2_0 = fluid_2_0 || {};
             tocHeader: "Table of Contents"
         },
         selectors: {
-            headings: ":header:visible:not(.flc-toc-tocContainer :header)",
+            headings: ":header:visible:not(.flc-toc-tocContainer :header, .flc-toc-exclude :header, .flc-toc-exclude)",
```

Comment:
See earlier comment - I think this feature would be more usable if it was possible for users to freely contribute "things to ignore" into a map, rather than having to unpick the logic of a single giant selector. Unfortunately the configuration "selectorsToIgnore" is taken already : P I suggest something like 

ignoreForToC: {
   overview: ".flc-overviewPanel",
   key2: "mySelector2"
}
etc
The implementation would then collect together all the values in this structure and ignore them collectively.